src/alchemy/preact/Cooldown.tsx

```diff
@@ -54,13 +54,17 @@ export class CoolDown extends Component<
           cActive: this.state.ttl >= 0,
           progress: !!this.props.progress,
         })}
+        role="status"
```

Comment:
I think the [progressbar role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role) would actually be more appropriate here

```suggestion
        role="progressbar"
        aria-valuenow={this.state.progress}
        aria-valuemax={0}
        aria-valuemin={1}
        aria-valuetext={`Cooling down for ${this.state.ttl} seconds`} 
```